packages/cli/generators/service/index.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+// no translation: Service
```

Comment:
Same comment here. I see several `g.f` calls below that contains words we may not want to translate (`datasource`, `service`, etc.).